src/CalendarApp.Domain/Services/MeetingRemoverService.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CalendarApp.Contracts;
+using CalendarApp.DataAccess.Repositories.Interfaces;
+using CalendarApp.Domain.Exceptions;
+using CalendarApp.Domain.Services.Interfaces;
+
+namespace CalendarApp.Domain.Services;
+
+public class MeetingRemoverService : IMeetingRemoverService
+{
+    public string DeleteMeeting(string meetingName)
```

Comment:
better to just return a number of removed lines, developers would not analyze the string to understand how many meetings were removed. for user interface it's better to create string in presenter